defaults/main.yml

```diff
@@ -63,6 +67,7 @@ django_pip_paths:
 django_pip_packages:
```

Comment:
question: @evansmurithi instead of having two pip package lists (`django_dependency_pip_packages` and `django_pip_packages`), what do you think the consequence of moving the task to install `django_pip_packages` higer up the list of tasks (to probably where the new task to install `django_dependency_pip_packages` is) will be?

Reply:
@jasonrogena thought of that but saw someone may need to upgrade packages installed after the requirements packages have been installed. If we moved the task before, the requirements file task would overwrite the package versions.

Reply:
Cool